### DISCUSSION TITLE
Astro naming collision, confusing! (United States Space Force, Missile System Center, Enterprise Ground Services, Satellite Operations)

### DISCUSSION THREAD

**Discussion by danielweck** [withastro/docs#2680] [Ideas]
Just a "FYI", not much can be done about this naming collision:
https://www.astrouxds.com/getting-started/

...ironically, the "Astro design system" website is built with ... Astro! :)

https://github.com/RocketCommunicationsInc/astro-www/
=>
https://astro-astro-www.netlify.app/getting-started/

I see that @jonathantneal is involved in the Astro UXDS website implementation.
(he is active in the Astro community, notably: https://github.com/astro-community/flow )

Side note: the UI components themselves are available for different frontend libs (it's just the above website that's made with Astro): https://astro-components.netlify.app

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

Indeed, that's a lot of Astro! 😄 
But it seems we can't do anything in docs, so I'm closing this to keep only relevant discussions opened.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
